scripts/release/generate-changelog.js

```diff
@@ -73,53 +75,56 @@ function getChangesList() {
   const lastTagName = command`git describe --tags ${lastTagHash}`.run()
 
   const commits = command`git log ${lastTagName.trimEnd()}..HEAD --pretty=format:%s`.run()
-
   const changesWithEmojis = emojiNameToUnicode(commits)
 
-  const allowedChanges = changesWithEmojis
-    .split('\n')
-    .filter(isNotVersionEntry)
-    .filter(isNotMaintenanceEntry)
-    .sort(byEmojiPriority)
+  let changes = changesWithEmojis.split('\n').filter(isNotVersionEntry)
+  let internalChanges = []
+  let publicChanges = []
+
+  changes.forEach((entry) => {
+    let trimmedEntry = entry.trim()
+    if (INTERNAL_EMOJI_PRIORITY.some((emoji) => trimmedEntry.startsWith(emoji))) {
+      internalChanges.push(entry)
+    } else {
+      publicChanges.push(entry)
+    }
+  })
+
+  internalChanges = internalChanges
+    .sort((a, b) => sortByEmojiPriority(a, b, INTERNAL_EMOJI_PRIORITY))
+    .map((entry) => `- ${entry}`)
+  publicChanges = publicChanges
+    .sort((a, b) => sortByEmojiPriority(a, b, PUBLIC_EMOJI_PRIORITY))
     .map((entry) => `- ${entry}`)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** Could be simplified: 
‏
```suggestion
const sortAndFormat = (entries, priority) =>
  entries
    .sort((a, b) => sortByEmojiPriority(a, b, priority))
    .map((entry) => `- ${entry}`);

internalChanges = sortAndFormat(internalChanges, INTERNAL_EMOJI_PRIORITY);
publicChanges = sortAndFormat(publicChanges, PUBLIC_EMOJI_PRIORITY);
```